support/src/testFixtures/java/se/inera/intyg/common/support/facade/testsetup/model/value/ValueCodeTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2022 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.common.support.facade.testsetup.model.value;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueCode;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueType;
+
+public abstract class ValueCodeTest extends ValueTest {
+
+    @Override
+    protected CertificateDataValueType getType() {
+        return CertificateDataValueType.CODE;
+    }
+
+    protected abstract CertificateDataValueCode getCertificateDataValueCode();
+
+    @Test
+    void shouldIncludeCertificateDataValueCode() {
```

Comment:
Does it make sense to break this function in to two separate tests? One that checks id & one that checks code:


 @Test
    void shouldIncludeCode() {
        final var value = (CertificateDataValueCode) getElement().getValue();
        assertEquals(getCode(), value.getCode());
    }


    @Test
    void shouldIncludeCodeId() {
        final var value = (CertificateDataValueCode) getElement().getValue();
        assertEquals(getCodeId(), value.getId());
    }